<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>338</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>338</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>338</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_photo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>37</y>
     <width>62</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">min-width:60px;
min-height:60px;
max-width:60px;
max-height:60px;
border-radius:30px;
border:1px solid black;

</string>
   </property>
   <property name="text">
    <string>头像</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>338</width>
     <height>768</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>338</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>338</width>
     <height>768</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/base/个人信息展示图片.jpg</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_change">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>680</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_quit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>680</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background:transparent;
border-width:0;
border-style:outset
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_sex">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>女</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_birth">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>390</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>2000.8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_age">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>450</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>18</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_zod">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>510</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>狮子座</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_loti">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>565</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 11pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>重庆</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_signa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;等线&quot;;</string>
   </property>
   <property name="text">
    <string>交个朋友吧~</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>label_user</zorder>
  <zorder>label_photo</zorder>
  <zorder>btn_change</zorder>
  <zorder>btn_quit</zorder>
  <zorder>label_sex</zorder>
  <zorder>label_birth</zorder>
  <zorder>label_age</zorder>
  <zorder>label_zod</zorder>
  <zorder>label_loti</zorder>
  <zorder>label_signa</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
